src/main/java/com/example/cart_19_04_22/service/impl/CartServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.example.cart_19_04_22.service.impl;
+
+import com.example.cart_19_04_22.modl.Cart;
+import com.example.cart_19_04_22.service.CartService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CartServiceImpl implements CartService {
+    private Cart cart;
```

Comment:
```suggestion
    private final Cart cart;
```
так как поле не предполагает переопределение значения